### PR TITLE
docs: Document path excludes

### PR DESCRIPTION
**Pull request by mnonnenmacher**
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1378)

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -32,6 +32,28 @@ explanation consists of:
...
+  - pattern: "test-data/**"
+    reason: "TEST_TOOL_OF"
+    comment: "This folder contains examples which are not distributed."
+```
+
+The path exclude above has the following effects:
+
+* All projects found below the `test-data` directory are marked as excluded.
+* License findings in files below the `test-data` directory are marked as excluded. This can be used to ignore such
+  license findings when writing [evaluator rules](GettingStarted.md#6-running-the-evaluator).
```
Comment (sschuberth) [inline docs/Configuration.md]: I'm not sure I fully get what you intend to say in relation to evaluator rules here. Do you want to say

> This mechanism can be used to easily ignore license findings in evaluator rules by checking whether they are excluded.

?

File: docs/Configuration.md [hunk 2]
```diff
@@ -32,6 +32,28 @@ explanation consists of:
 
 The sections below contain links to the lists of available exclude reasons for each type of exclude.
 
+#### Excluding Paths
+
+Path excludes are used to mark a complete path as excluded. They are defined using a
+[glob pattern](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob), a `reason` and a `comment`:
```
Comment (sschuberth) [inline docs/Configuration.md]: I believe we should also explain the difference to project excludes here, or even clearly say that path excludes are a more generic replacement for project excludes, and that the latter are deprecated.

File: docs/Configuration.md (hunk 1 above)
Comment (mnonnenmacher) [reply docs/Configuration.md]: Done.

File: docs/Configuration.md (hunk 2 above)
Comment (mnonnenmacher) [reply docs/Configuration.md]: I have added a comment about this. However, let's first discuss offline how we want to implement project specific scope excludes, before marking project excludes as deprecated.

**Pull request merged**